components/console-backend-service/internal/gqlschema/schema.graphql

```diff
@@ -833,6 +838,66 @@ input APIInput {
     authenticationEnabled: Boolean
 }
 
+# API Gateway Controller
+
+type APIRule {
+    name: String!
```

Comment:
Question: Isn't namespace necessary?

Reply:
IMHO it is not. As the namespace has to be provided to fetch resources, the user already knows a namespace of the resource.